test-graalvm/common/src/main/java/testgraalvm/repositories/OwnerRepository.java

```diff
@@ -16,6 +15,4 @@ public interface OwnerRepository extends ReactorCrudRepository<Owner, Long> {
     Mono<OwnerDto> findByName(String name);
 
     Mono<OwnerDto> getByName(String name);
-
```

Comment:
@dstepanov This was throwing error after switching to micronaut-data 4.2.0, probably related to refactoring in method matcher in 4.2.0. The error was

```
:micronaut-test-graalvm:micronaut-h2:compileJava FAILED |  
-- | --
  | error: Unable to implement Repository method: H2OwnerRepository.saveAll(Iterable entities,Connection connection). Cannot save with non-existent property or constructor argument: entities |  
  | Note: Creating bean classes for 2 type elements |  
  | 1 error
```
and when `Connection connection` parameter is removed then it works. It's not an issue for the tests, not sure how connection was used or propagated anyway (as rule parameter?) but it could break user's apps that could have been using this.

Reply:
Changed to `ReactiveTransactionStatus<Connection>` which is supported as role parameter and it works ok then.